.github/workflows/qemu.yml

```diff
@@ -0,0 +1,127 @@
+name: qemu
+on: [push]
+jobs:
+  build:
+    name: Build OpenWrt
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+       include:
+         - name: armvirt-64
+         - name: malta-32
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install \
+            build-essential \
+            ccache \
+            clang-12 \
+            ecj \
+            fastjar \
+            file \
+            g++ \
+            gawk \
+            gettext \
+            git \
+            java-propose-classpath \
+            libelf-dev \
+            libncurses-dev \
+            libssl-dev \
+            mkisofs \
+            python3 \
+            python3-dev \
+            python3-distutils \
+            python3-setuptools \
+            qemu-utils \
+            rsync \
+            subversion \
+            swig \
+            unzip \
+            wget \
+            xsltproc \
+            zlib1g-dev
+
+      - name: Create config (armvirt-64)
+        if: ${{ matrix.name == 'armvirt-64' }}
+        run: |
+          echo CONFIG_TARGET_armvirt=y >> .config
+          echo CONFIG_TARGET_armvirt_64=y >> .config
+          echo CONFIG_TARGET_armvirt_64_Default=y >> .config
+          echo CONFIG_FEED_luci=y >> .config
+          echo CONFIG_FEED_packages=y >> .config
+          echo CONFIG_FEED_routing=y >> .config
+          echo CONFIG_FEED_telephony=y >> .config
+
+      - name: Create config (malta-32)
+        if: ${{ matrix.name == 'malta-32' }}
```

Comment:
Instead of this if I would prefer to add the list of configuration options to the matrix configuration, but I was unable to add a multi line list there and then echo it out here. Does anyone have an idea?

Reply:
A simpler approach is to split the matrix values target/subtarget into env variables, see [here](https://github.com/aparcar/openwrt/blob/apk/.github/workflows/build-staging.yml#L144).

This way you could do something like the following:

```shell
TARGET=$(echo ${{ matrix.target }} | cut -d "/" -f 1)
SUBTARGET=$(echo ${{ matrix.target }} | cut -d "/" -f 2)
echo CONFIG_TARGET_${TARGET}=y >> .config
echo CONFIG_TARGET_${TARGET}_${SUBTARGET}=y >> .config
```

Reply:
I'm using `/` as separator but you can really use anything you like.